src/v2/Components/LocationAutocompleteInput.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+import { DEFAULT_MODAL_Z_INDEX, Input, InputProps } from "@artsy/palette"
+import { useLoadScript } from "v2/Utils/Hooks/useLoadScript"
+import { data as sd } from "sharify"
+import { useRef } from "react"
+import { useEffect } from "react"
+import { useState } from "react"
+import { useAppendStylesheet } from "v2/Utils/Hooks/useAppendStylesheet"
+
+const GOOGLE_PLACES_API_SRC = `https://maps.googleapis.com/maps/api/js?key=${sd.GOOGLE_MAPS_API_KEY}&libraries=places&language=en`
+
+interface LocationAutocompleteInputProps extends Omit<InputProps, "onChange"> {
+  onChange(place: Place): void
+}
+
+export const LocationAutocompleteInput: React.FC<LocationAutocompleteInputProps> = ({
+  onChange,
+  ...rest
+}) => {
+  const [ready, setReady] = useState(false)
+
+  useLoadScript({
```

Comment:
Can we do a follow-up and audit force for other locations where similar functionality is being used and update? I'm thinking the recaptcha component.

Reply:
:waits for @damassi to scroll down: 😀

Reply:
hah